src/style.css

```diff
@@ -1,3 +1,132 @@
 body {
   margin: 0;
+  display: block;
+  padding: 0;
+  box-sizing: border-box;
+
+  overflow-x: hidden;
+}
+
+.header {
+  width: 100%;
+  height: 60px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+  margin: 0 auto;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  top: 10px;
+  left: 50px;
+}
+
+.logo_image {
+  display: block;
+  width: 100%;
+  height: 100%;
+}
+
+.nav {
+  display: block;
+}
+
+.moyo_menu {
+  display: flex;
+  list-style-type: none;
+}
+
+.menu-item {
+  margin-right: 30px;
+}
+
+.menu-item:first-child {
+  margin-left: 0;
+}
+
+.menu-item:last-child {
+  margin-left: 0;
+  padding-right: 50px;
+}
+
+.menu_link {
+  position: relative;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  font-family: Roboto, sans-serif;
+  line-height: 14.06px;
+  font-size: 12px;
+  font-weight: 500;
+  color: #000;
+  white-space: nowrap;
+  text-decoration: none;
+  text-transform: uppercase;
+  transition: color 0.3s;
+}
+
+.menu_link:hover {
+  text-decoration: underline;
+}
+
+a.menu_link.is-active {
+  color: #00acdc;
+}
+
+a.is-active::after {
+  content: '';
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  height: 4px;
+  border-radius: 2px;
+  background: #00acdc;
+  transition: width 0.3s ease;
+}
+
+[data-qa='hover'] {
+  color: #000;
+}
+
+[data-qa='hover']:hover {
+  color: #00acdc;
+  text-decoration: none;
+}
+
+@media (min-width: 1200px) {
+  .header {
+    width: 37px;
+    height: 15px;
+    top: 22.5px;
+    left: 509px;
+  }
+
+  .nav {
```

Comment:
Similar to the comment for `.header`, the dimensions set for `.nav` within the media query at line 102 (lines 111 and 112) seem incorrect. Ensure that the dimensions are suitable for the content and layout requirements at different screen sizes.